src/db/SQL.js

```diff
@@ -402,7 +402,7 @@ define([
 				return results;
 			}));
 			var store = this;
-			return new QueryResults(results, {
+			return QueryResults.default(results, {
```

Comment:
I would probably prefer that for the time being assuming TS doesn't gripe about it, but it's kind of a toss-up right now and I won't ask you to do it unless it doesn't get in your way or you don't have better things to do.